crates/hc_bundle/tests/test_cli.rs

```diff
@@ -283,6 +284,29 @@ async fn test_multi_integrity() {
     assert_eq!(*dna.dna_def(), expected);
 }
 
+#[tokio::test]
+async fn test_hash_dna_function() {
+    {
+        let mut cmd = Command::cargo_bin("hc-dna").unwrap();
+        let cmd = cmd.args(["hash", "tests/fixtures/my-app/dnas/dna1"]);
+        cmd.assert().success();
+    }
+    {
+        let mut cmd = Command::cargo_bin("hc-dna").unwrap();
+        let cmd = cmd.args(["hash", "tests/fixtures/my-app/dnas/dna1/a dna.dna"]);
+        cmd.assert().success();
+    }
+    {
+        let mut cmd = Command::cargo_bin("hc-dna").unwrap();
+        let cmd = cmd.args(["hash", "tests/fixtures/my-app/dnas/dna1/a dna.dna"]);
+        let original =
+            DnaHashB64::from_b64_str("uhC0klkazCjMK-V3HooCgXVCB7OGhGEplGD-UWFgCIeXGZfRB7ORO")
```

Comment:
Leaving a note for other reviewers, this DNA is not built from real wasms that include the HDI/HDK so this DNA hash isn't expected to change with code changes. It's valid to match directly on it

Reply:
as you can see this test is failing for windows with a different hash (Linux and Mac pass) 
Expected: uhC0klkazCjMK-V3HooCgXVCB7OGhGEplGD-UWFgCIeXGZfRB7ORO
Actual: uhC0kakHM3GmBlqkkTXp6Pm0ELJxqfwiOujuy0PRbY83OAk6YNipk

this i suspect is because either the hashing function or the file byte decoding on windows adds chars / works differently. investigation needed

Reply:
I would prefer not to merge the PR without this test. Can I suggest that instead the test is disabled on Windows with a note about why? Ideally pointing to an issue please?

Reply:
We have several examples around of tests that don't behave right on other platforms. Something like `cfg_attr(ignore = "flake on mac", macos)`. Can't think exactly what that looks like, but along those lines